project/Core/Inc/flash_interface.h

```diff
@@ -8,26 +8,58 @@
 #include <string.h>
 #include <stdio.h>
 
+// Flash sector definitions
+#define FLASH_SECTOR_CAMERA     1
+#define FLASH_SECTOR_2          2  // Camera overflow
+#define FLASH_SECTOR_3          3  // Camera overflow
+#define FLASH_SECTOR_SENSORS    4
+#define FLASH_SECTOR_ALTIMETER  5
+#define FLASH_SECTOR_GPS        6
+#define FLASH_SECTOR_BATTERY    7
```

Comment:
If these were defined as unsigned ints and passed in place of the hal sector definitions, could get_hal_sector be removed?
i.e.
```suggestion
#define FLASH_SECTOR_CAMERA     1U
#define FLASH_SECTOR_2          2U  // Camera overflow
#define FLASH_SECTOR_3          3U // Camera overflow
#define FLASH_SECTOR_SENSORS    4U
#define FLASH_SECTOR_ALTIMETER  5U
#define FLASH_SECTOR_GPS        6U
#define FLASH_SECTOR_BATTERY    7U
```